docker/templates/chain-monitor-config.json

```diff
@@ -14,17 +14,19 @@
           "dai_gateway": "0x0000000000000000000000000000000000000000",
           "usdc_gateway": "0x0000000000000000000000000000000000000000",
           "lido_gateway": "0x0000000000000000000000000000000000000000",
-          "puffer_gateway": "0x0000000000000000000000000000000000000000"
+          "puffer_gateway": "0x0000000000000000000000000000000000000000",
+          "gas_token_gateway": null
         },
         "scroll_messenger": null,
         "message_queue": null,
-        "scroll_chain": null
+        "scroll_chain": null,
+        "gas_token": null
     },
-    "start_messenger_balance": null
+    "start_messenger_balance": 0
```

Comment:
I just realized it's missing. Added a draft here: https://github.com/scroll-tech/scroll-contracts/pull/24 probably needs some adjustments.

Reply:
I think contracts don't need chain_monitor config,  also the bridge-history-config.json

Reply:
> I think contracts don't need chain_monitor config, also the bridge-history-config.json

Yes, this is mainly because we want to have a single command to generate all config files for scroll-sdk, so here the file https://github.com/scroll-tech/scroll-contracts/blob/feat-deployment-alt-gas-token/docker/scripts/gen-configs.sh will help to do so.

Reply:
i see. but i don't like this way. why not generate the configs in scroll-sdk repo?

Reply:
Yes, this is a convenient way to generate the configs before actually deploying the contracts. But I'm fine with moving the `GenerateConfigs` logic into `scroll-sdk`, or maybe add it to Daniel's cli.

Reply:
> Why 0? L1 messenger balance will not be 0 in a real deployment.

For this, my original thinking is to burn rest of the deployer balance to all zero address, so we can set chain monitor `start_messenger_balance` to 0. but in this way L2 contracts deployment gas fee may not be burned.

but with https://github.com/scroll-tech/scroll-contracts/pull/24 , would be better.